charts/redpanda/Chart.yaml

```diff
@@ -23,7 +23,7 @@ type: application
 # The chart version and the app version are not the same and will not track
 # together. The chart version is a semver representation of changes to this
 # chart.
-version: 5.0.4
+version: 5.0.7
```

Comment:
NIT: Why such bump?

From 5.0.4 to 5.0.7 seems 3 patch version jump.

Reply:
I was counting on a few of your commits to merge at the time i made that change. Ill update in a few.